content/blog/announcing-kustomize-support/index.md

```diff
@@ -0,0 +1,340 @@
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/kustomize"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		_, err := kustomize.NewDirectory(ctx, "helloWorldLocal",
+			kustomize.DirectoryArgs{
+                Directory: pulumi.String("./helloWorld"),
+            },
+		)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+![Kustomize demo](kustomize.gif)
+
+## Working with git repos
+
+We can also use Kustomizations directly from a git repository, local or remote,
+by specifying the path to the directory.
+
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import * as k8s from "@pulumi/kubernetes";
+
+new k8s.kustomize.Directory("helloWorldRemote", {
+    directory: "https://github.com/kubernetes-sigs/kustomize/tree/v3.3.1/examples/helloWorld",
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi_kubernetes as k8s
+
+k8s.kustomize.Directory("helloWorldRemote",
+                        directory="https://github.com/kubernetes-sigs/kustomize/tree/v3.3.1/examples/helloWorld")
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using System;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.Kubernetes.Kustomize;
+
+class KustomizeStack : Stack
+{
+    public KustomizeStack()
+    {
+        var files = new Directory("helloWorldRemote", new DirectoryArgs
+        {
+            Directory = "https://github.com/kubernetes-sigs/kustomize/tree/v3.3.1/examples/helloWorld"
+        });
+    }
+
+}
+
+class Program
+{
+    static Task<int> Main(string[] args) => Deployment.RunAsync<KustomizeStack>();
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/kustomize"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		_, err := kustomize.NewDirectory(ctx, "helloWorldRemote",
+			kustomize.DirectoryArgs{
+                Directory: pulumi.String("https://github.com/kubernetes-sigs/kustomize/tree/v3.3.1/examples/helloWorld"),
+            },
+		)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+## A better way to Kustomize
+
+One of my favorite parts of Pulumi's Kubernetes SDK is the ability to apply transformations to resource definitions.
+This feature allows you to make changes to existing definitions before they are applied to a cluster. This saves
+the headache of forking an upstream repo just to make a few tweaks, and then trying to keep those changes in sync!
+Transformations are available for the Helm, Kustomize, and YAML SDKs.
```

Comment:
Consider adding a note about kustomize overlays and how those are framed differently compared to transformations since both accomplish similar goals.

Reply:
Done!